### PR TITLE
check: update instruction for auto-increment key checking

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
we decide to remove the column mapping, so need update the instruction

### What is changed and how it works?
update instruction

### CODE REVIEW

File: pkg/check/table_structure.go [hunk 1]
```diff
@@ -374,7 +374,7 @@ func (c *ShardingTablesCheck) checkAutoIncrementKey(instance, schema, table stri
 		if !hasMatchedRule {
 			r.State = StateFailure
 			r.ErrorMsg = fmt.Sprintf("instance %s table `%s`.`%s` of sharding %s have auto-increment key, would conflict with each other to cause data corruption", instance, schema, table, c.name)
-			r.Instruction = "please set column mapping rules for them, or handle it by yourself"
+			r.Instruction = "please handle it by yourself, read https://pingcap.com/docs-cn/dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard/#自增主键冲突处理 for more detail"
```
Comment (IANTHEREAL) [inline pkg/check/table_structure.go]: also put English document link here？

Comment (WangXiangUSTC) [reply pkg/check/table_structure.go]: don't have english doc now, update it later?

Comment (IANTHEREAL) [reply pkg/check/table_structure.go]: add english doc firstly, some foreign users need it

Comment (WangXiangUSTC) [reply pkg/check/table_structure.go]: ok

Comment (csuzhangxc) [reply pkg/check/table_structure.go]: should we link to `dev` or `v3.0`?

Comment (csuzhangxc) [reply pkg/check/table_structure.go]: If the user want to _continue_ after _handled it by themself_, what should they do in _dmctl_?

Comment (WangXiangUSTC) [reply pkg/check/table_structure.go]: because dm's version is independent, so I think dev is better.

Comment (WangXiangUSTC) [reply pkg/check/table_structure.go]: also add a tip for how to do? but this is a pkg, infact I think add a dm's link here is not very suitable.

**Comment by WangXiangUSTC** [comment]
it will take some days for i18n to translate the document to english, and I think `check` is a pkg, should not contain some information with DM. So how about just remove the tip about column mapping in `check`, and add some tip in DM? @csuzhangxc @GregoryIan

**Comment by csuzhangxc** [comment]
> it will take some days for i18n to translate the document to english, and I think `check` is a pkg, should not contain some information with DM. So how about just remove the tip about column mapping in `check`, and add some tip in DM? @csuzhangxc @GregoryIan

also remove tips for other pkgs?

**Comment by WangXiangUSTC** [comment]
I had checked other check item, their instruction are common, don't concern about specified project. @csuzhangxc

**Comment by csuzhangxc** [comment]
> I had checked other check item, their instruction are common, don't concern about specified project. @csuzhangxc

can we add a common instruction in the pkg, but overwrite it in DM?

**Comment by WangXiangUSTC** [comment]
yes, now all check items' result have common instruction. @csuzhangxc

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by IANTHEREAL — APPROVED** [review]
LGTM

**Pull request merged**